notebooks/quickstart.ipynb

```diff
@@ -120,18 +122,115 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "--------------------------------\n",
+    "## The basics of `spaghetti`\n",
+    "### 1. Creating a network instance\n",
+    "\n",
+    "Within `spaghetti` network objects can be created from [a variety of objects](https://pysal.org/spaghetti/generated/spaghetti.Network.html#spaghetti-network), the most common being shapefiles (read in as file paths) and `geopandas.GeoDataFrame` objects. However, a network could also be created from `libpysal` geometries, as demonstrated in the [connected components tutorial](https://pysal.org/spaghetti/notebooks/connected-components.html#1.-Instantiate-a-network-from-two-collections-of-libpysal.cg.Chain-objects) or a simply as follows:\n",
+    "\n",
+    "```python\n",
+    "from libpysal.cg import Point, Chain\n",
+    "import spaghetti\n",
+    "# create the network\n",
+    "ntw = spaghetti.Network(in_data=Chain([Point([1, 1]), Point([2, 1])]))\n",
+    "```\n",
+    "\n",
+    "This will create a single-segment network.\n",
```

Comment:
quick explanation/ definition of single-segment network

Reply:
Done. See [here](https://github.com/pysal/spaghetti/pull/603/commits/0fa1c8be881ab2deae7d8262a6e1a1664bdc970f#diff-25b99c166f43d62ac5585da3ae0be498de4716c491b74e0d6fb028466b4343edR138).